app/presenters/tree_builder_services.rb

```diff
@@ -23,13 +23,26 @@ def root_options
   def x_get_tree_roots(count_only, _options)
     roots = Rbac.filtered(Service.roots)
 
+    # Define array of roots with children
+    coalesce_func = Arel::Nodes::NamedFunction.new(
+      "COALESCE",
+      [Service.arel_table[:ancestry], Arel::Nodes::SqlLiteral.new("'0'")]
```

Comment:
I wish ancestry had this build in. Very tired of dealing with ancestry.nil? and having to use 2 different columns.